03.asynchronous/callback.js

```diff
@@ -0,0 +1,48 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory:");
+
+function callback() {
+  db.run(
+    "CREATE TABLE IF NOT EXISTS test_table(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+    function () {
+      db.run(
+        "INSERT INTO test_table(title) values(?)",
```

Comment:
SQL のキーワードで大文字と小文字が混在しています。大文字に統一したほうがよいと思います。

```suggestion
        "INSERT INTO test_table(title) VALUES(?)",
```

Reply:
すみません！漏れでした🙏
大文字に統一しました。